content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,45 @@
+
+These sorts of temptations often arise because a large number of residents of a state could benefit from them. Thus, they can be hard for the state government to resist. A national government made up of diverse states with different types of citizens will avoid these temptations and will therefore do better at keeping its treaties.
+
+So, as far as intentional or accidental violation of treaties is a cause for war, a national government will be better at keeping treaties than several smaller governments would be, and will therefore make us safer.
+
+When it comes to wars from direct and unlawful attacks, it is at least as clear to me that a single national government is vastly more secure against this kind of danger.
+
+These kinds of wars are usually caused by the passions of a faction (e.g., one or two states). For example, none of the wars with the American Indians have come about because of aggressions by the federal government<sup>[2](#footnote2)</sup>. Rather, these hostilities have been the result of provocations by individual states who were either unwilling or unable to punish wrongdoers, causing the slaughter of many innocent American Indians.
+
+This dynamic is most dangerous on our borders with Spanish and British territories. These bordering states might be tempted to lash out after a quarrel or irritation, dragging us into war with one of these powerful nations. Nothing can help to avoid this danger more than a national government, where decisions can be made outside of the context of the passions of the moment.
+
+Not only will a national government give fewer just causes of war, but it will also be better able to settle wars when they do happen. States, like men, are always trying to justify their actions rather than acknowledging and correcting errors or offenses. The national government will not be affected by this pride, but will act in moderation and candor to identify means to avoid and escape from difficulties.
+
+In addition, being united gives us a much stronger bargaining position. For example, in 1865 Louis XIV made Genoa send their chief magistrate and four senators to France to grovel before him. The Genoans had to do it for the sake of peace, but can you imagine Louis XIV demanding (or receiving) this humiliation from a powerful opponent like Spain or Britain?
+
+
+---
+<a name="footnote1">Footnote 1</a>: Not *every* prediction made by the founders has come true.
+<a name="footnote2">Footnote 2</a>: This prediction was devastatingly wrong and the federal government was involved with [aggressions against American Indians](https://en.wikipedia.org/wiki/American_Indian_Wars) for more than a century.
```

Comment:
This was not a prediction, but a statement on the past. Maybe he's exaggerating, but this was probably true at the time. It may even be close to true after that. The federal government was involved in much of the wrong, but the states often provoked and initiated it.

Reply:
It feels like we need some acknowledgement that the federal government ended up perpetrating atrocities against Native Americans. I'll try another tongue-in-cheek option in the text, but I bet you'll like it less. :)

Reply:
Meh, I didn't like that idea either. Anyway, I don't know how to handle this.